EIPS/eip-2537.md

```diff
@@ -320,7 +333,8 @@ There are no backward compatibility questions.
 
 ### Subgroup checks
 
-A subgroup check **is mandatory** during the pairing call. Implementations *should* use fast subgroup checks: at the time of writing, multiplication gas cost is based on the `double-and-add` multiplication method that has a clear "worst case" (all bits are equal to one). For pairing operations, it is expected that implementations use faster subgroup checks, e.g. by using the wNAF multiplication method for elliptic curves that is ~ `40%` cheaper with windows size equal to 4. (Tested empirically. Savings are due to lower hamming weight of the group order and even lower hamming weight for wNAF. Concretely, subgroup check for both G1 and G2 points in a pair are around `35000` combined).
+The subgroup check for point `H` verifies that `H` multiplied by scalar `q` yields the infinity point on the respective curve. 
```

Comment:
rather than use this naive expensive subgroup check (in particular for G2) I'd rather suggest to point to https://eprint.iacr.org/2021/1130.pdf

Reply:
Can we provide complexity estimates for this method? What is `u`?

Reply:
it's all in the paper. 'u' is the seed for generating BLS12-381 and pretty short ( -0xd201000000010000)

Reply:
Reformulated the text with your link @asanso